### PR TITLE
AE-436 Aave Governance DAO Events

### PR DESCRIPTION
**Pull request by arbitrarylink**
Adds the following events from the associated contracts to scan.

Events per contract:

AaveGovernanceV2:  0xEC568fffba86c094cf06b22134B23074DFE2252c
ExecutorAuthorized
ExecutorUnauthorized
GoveranceStrategyChanged
OwnershipTransferred
ProposalCanceled
ProposalCreated
ProposalExecuted
ProposalQueued
VoteEmitted
VotingDelayChanged

Executor (short)
0xee56e2b3d491590b5b31738cc34d5232f378a8d5
Executor (long):
0x61910ecd7e8e942136ce7fe7943f956cea1cc2f7
CancelledAction
ExecutedAction
NewAdmin
NewDelay
NewPendingAdmin
QueuedAction

Ecosystem Reserve
0x25F2226B597E8F9514B3F68F00f494cF4f286491
NewFundsAdmin

### CODE REVIEW

File: contract-addresses.json [hunk 1]
```diff
@@ -11,5 +11,9 @@
   "ProtocolDataProvider" : "0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d",
   "WETHGateway" : "0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04",
   "AaveCollector" : "0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c",
-  "IncentivesController" : "0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5"
+  "IncentivesController" : "0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5",
+  "AaveGovernanceV2" : "0xEC568fffba86c094cf06b22134B23074DFE2252c",
+  "ExecutorShort" : "0xee56e2b3d491590b5b31738cc34d5232f378a8d5#readContract",
+  "ExecutorLong" : "0x61910ecd7e8e942136ce7fe7943f956cea1cc2f7",
+  "EcosystemReserve" : "0x25f2226b597e8f9514b3f68f00f494cf4f286491"
```
Comment (y0wl) [inline contract-addresses.json:18]: these addresses aren't checksummed. Also you copied the #readcontract from the URL in etherscan. might be best to press the copy button to get the checksummed addresses

File: src/admin-events/admin-events.json [hunk 2]
```diff
@@ -31,7 +31,35 @@
...
+    ],
+  "ExecutorLong" : [
+    "CancelledAction(bytes32,address,uint256,string,bytes,uint256,bool)",
+    "ExecutedAction(bytes32,address,uint256,string,bytes,uint256,bool,bytes)",
+    "NewAdmin(address)",
+    "NewDelay(uint256)",
+    "NewPendingAdmin(address)"
+    ],
+  "EcosystemReserve" : [
+    "NewFundsAdmin(address)"
```
Comment (y0wl) [inline src/admin-events/admin-events.json:63]: not your fault, but the indentation in this file is messed up. I know in vim you can fix indentation with `<esc>gg=G`

**Review by y0wl — CHANGES_REQUESTED** [review]

File: contract-addresses.json (hunk 1 above)
Comment (arbitrarylink) [reply contract-addresses.json]: Fixed

File: src/admin-events/admin-events.json (hunk 2 above)
Comment (arbitrarylink) [reply src/admin-events/admin-events.json]: Fixed by using <esc>gg=G in VIM.

**Review by arbitrary-cr0 — APPROVED** [review]

File: src/admin-events/admin-events.json [hunk 3]
```diff
@@ -1,37 +1,65 @@
...
+		"CancelledAction(bytes32,address,uint256,string,bytes,uint256,bool)",
+	"ExecutedAction(bytes32,address,uint256,string,bytes,uint256,bool,bytes)",
+	"NewAdmin(address)",
+	"NewDelay(uint256)",
+	"NewPendingAdmin(address)"
+	],
+	"EcosystemReserve" : [
+		"NewFundsAdmin(address)"
+	]
 }
```
Comment (y0wl) [inline src/admin-events/admin-events.json:65]: tabs -> 2 spaces
there's vim ways to do this I'm sure. 
I have this in my vimrc
```" language specific tabbing
autocmd FileType javascript setlocal shiftwidth=2 tabstop=2
autocmd FileType json setlocal shiftwidth=2 tabstop=2
```

**Review by arbitrary-kdawg — APPROVED** [review]

**Pull request merged**